installer/app/src/views/azure-create-app-tutorial.js.jsx

```diff
@@ -5,163 +5,240 @@ import Sheet from './css/sheet';
-						}} />" as the "REDIRECT URI"</p>
-					<p>Click the checkmark in the bottom right to continue</p>
-				</li>
-
-				<li>
-					<img src={AssetPaths["azure-app-6.png"]}  style={{
-						width: 659,
-						height: 278
-					}}/>
-					<label>
-						<p>Click the "CONFIGURE" tab</p>
-						<p>Copy the "CLIENT ID" into the input below</p>
-						<input name="client_id" type="text" placeholder="CLIENT ID" />
-					</label>
-				</li>
-
-				<li>
-					<img src={AssetPaths["azure-app-7.png"]}  style={{
-						width: 659,
-						height: 155
-					}}/>
-					<p>Scroll to the bottom of the configuration page</p>
-					<p>Click the green "Add application" button</p>
-				</li>
-
-				<li>
-					<img src={AssetPaths["azure-app-9.png"]}  style={{
-						width: 659,
-						height: 415
-					}}/>
-					<p>Click the "Windows Azure Service ..." option</p>
-					<p>Click the checkmark in the bottom right to continue</p>
-				</li>
-
-				<li>
-					<img src={AssetPaths["azure-app-10.png"]}  style={{
-						width: 659,
-						height: 169
-					}}/>
-					<p>Click the "Delegated Permissions" dropdown</p>
-					<p>Check "Access Azure Service Management"</p>
-				</li>
-
-				<li>
-					<img src={AssetPaths["azure-app-11.png"]}  style={{
-						width: 659,
-						height: 93
-					}}/>
-					<p>Click the "Save" button</p>
-				</li>
-
-				<li>
-					<img src={AssetPaths["azure-app-12.png"]}  style={{
-						width: 659,
-						height: 486
-					}}/>
-					<label>
-						<p>Click on the back arrow button to go back to the "APPLICATIONS" tab click and the "ENDPOINTS" button at the bottom</p>
-						<p>Copy your OAuth 2.0 Token Endpoint into the input below</p>
-						<input name="endpoint" type="text" placeholder="https://login.microsoftonline.com/{your-uid}/oauth2/token?api-version=1.0" />
-					</label>
-				</li>
-			</ol>
-		);
-	},
-
-	componentDidMount: function () {
-		this.state.styleEl.commit();
-	},
-
-	__handleRedirectURIInputClick: function (e) {
-		InputSelection.selectAll(e.target);
-	}
+    getInitialState: function () {
+        var styleEl = Sheet.createElement({
+            marginTop: '1rem',
+            selectors: [
+                ['> li > img', {
+                    verticalAlign: 'text-top',
+                    border: '1px solid #333'
+                }],
+                ['input[data-selectable]', {
+                    backgroundColor: 'transparent',
+                    border: 'none',
+                    color: 'inherit',
+                    textAlign: 'center',
+                    padding: 0
+                }]
+            ]
+        });
+
+        return {
+            styleEl: styleEl
+        };
+    },
+
+    render: function () {
+        var redirectURI = window.location.protocol + '//'+ window.location.host + '/oauth/azure',
+            tutorialSlides = [''],
+            inputStyles = this.__getInputStyles(),
+            s = this.state,
+            intro, pagination = {}, tutorialSlide;
```

Comment:
Define all the vars with separate `var` statements (better alignment with tabs and clearer diff when making changes).